stm32cube/CMakeLists.txt

```diff
@@ -51,6 +51,10 @@ if(CONFIG_CLOCK_STM32_HSE_CLOCK)
   zephyr_compile_definitions( -DHSE_VALUE=${CONFIG_CLOCK_STM32_HSE_CLOCK} )
 endif()
 
+if(CONFIG_PTP_CLOCK_STM32_HAL)
+  zephyr_compile_definitions( -DHAL_ETH_USE_PTP)
+endif()
```

Comment:
Please make it a dedicated commit as this is different from enabling new eth HAL API.

Reply:
From what I can check this is only available with the new API, so dismissing this request.